compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -974,8 +973,8 @@ static TR::InstOpCode::Mnemonic cmp2branch(TR::ILOpCodes op, TR::CodeGenerator *
        case TR::lucmpne:
        case TR::fcmpne:
        case TR::dcmpne:
-       case TR::fcmpneu:
        case TR::dcmpneu:
+       case TR::fcmpneu:
```

Comment:
This doesn't technically change anything, but I don't see a need to move `fcmpneu`. :wink: